lib/jekyll/page.rb

```diff
@@ -155,11 +139,6 @@ def destination(dest)
       path
     end
 
-    # Returns the object as a debug String.
-    def inspect
```

Comment:
* WARNING: Jekyll holds very larges objects, removing a custom inspect can cause a large amount of data in REPL's.